### PR TITLE
Metabolomics second try

### PR DESCRIPTION
**Pull request by lecorguille**
So, let try again.
A long time ago, I did this: https://github.com/galaxyproject/training-material/pull/355

Since, our group ([Workflow4Metabolomics](http://workflow4metabolomics.org/)) decided to focus an Open Course base on Office Powerpoint 😞 
But, I believe that it will be easier to start from this Open Course then from our classic training support. The Open Course one are lighter as the GTN used to provide.

I will start with a tutorial (my first one)

### CODE REVIEW

**Comment by bgruening** [comment]
Thanks a bunch! This is super cool!

**Comment by lecorguille** [comment]
I'm done for today (and can't expect to find soon so many spare time to go further)

I consider this first tutorial enough mature to be review.
Not that the version of the tool I use are not available yet in the Main ToolShed but in the TestToolShed. I'm really close to release, that why I choose them.

If you read this tutorial "LC-MS - Data Import", you will surely find that I explain some trivial thing: upload the data, what is a Dataset collection... I agree but our user need that and especially those who used to use zip file as input.
The next tutorial or slide will be more scientific 😛 

Many thanks and bravo for those really good guidelines I follow today (even if I already proposed some materials in the past, it was some slides)

**Comment by bebatut** [comment]
Hi @lecorguille we have changed the way we organize metadata in this repository: I have updated your PR. The metadata are now at the top of the `tutorial.md` or `slides.html` files

**Comment by lecorguille** [comment]
@RJMW can you review (contain, spelling, ...) this little tutorial?
That way, we will be able to merge it and by the same way create the topics metabolomics... In which, you will be able to add a DIMSpy tuto.

**Comment by bebatut** [comment]
@lecorguille I disabled the tutorial: it will not appear in the topic but will still be accessible online (once merged).
We do that until the tools are on MTS (one of the requirements for the tutorials)

File: topics/metabolomics/slides/introduction.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Blabla
+
+- Blabla
+- Blabla
+
+Blabla
+
+- Blabla
+- Blabla
```
Comment (shiltemann) [inline topics/metabolomics/slides/introduction.html]: @lecorguille do you plan to fill these topic introduction slides as well? The slides are the only thing that will be visible in the topic until the tools for your tutorial are ready for the MTS, so some content in here would be nice. It doesn't have to be extensive, just couple slides about what metabolomics is would already be very helpful :)

Comment (lecorguille) [reply topics/metabolomics/slides/introduction.html]: Done or so

**Comment by lecorguille** [comment]
I have a question related to the contribution of people who aren't in GitHub as [here](https://github.com/galaxyproject/training-material/pull/958/files#diff-1cf3d9bed87e3621752e0d1d38741d87R8)
The last slide only show:

>This material is the result of a collaborative work. Thanks the Galaxy Training Network and all the contributors (Gildas Le Corguillé, ) !

File: topics/metabolomics/slides/introduction.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
+---
+layout: introduction_slides
+logo: "GTN"
+title: "Introduction to Metabolomics"
+type: "introduction"
+contributors:
+  - lecorguille
+  - Cécile Canlet - INRA - UMR Toxalim - France
```
Comment (bebatut) [inline topics/metabolomics/slides/introduction.html]: Can you add her to the `CONTRIBUTORS.yaml` file? With something like:

```
cecilecanlet:
    name: Cécile Canlet
    github: false
```

And replace here by `cecilecanlet`

Comment (lecorguille) [reply topics/metabolomics/slides/introduction.html]: Cool!
Done

**Comment by lecorguille** [comment]
@melpetera and @fgiacomoni Can you help me to give some "love" to this tutorial?

There is a bunch of **TODO** @bebatut added within the [tutorial.md](https://github.com/galaxyproject/training-material/blob/873568e21b172031b8db7309356fa18478bdee59/topics/metabolomics/tutorials/lcms-data-import/tutorial.md)

File: topics/metabolomics/docker/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Galaxy - metagenomics training material
+#
+# to build the docker image, go to root of training repo and
+#    docker build -t <your_tag> -f topics/<your_topic>/docker/Dockerfile .
```
[the remark this answers is not in the document]
Comment (erasche) [reply topics/metabolomics/docker/Dockerfile]:
```suggestion
#    docker build -t <your_tag> -f topics/metabolomics/docker/Dockerfile .
```

**Pull request merged**